support/fixtures/factories/facility.js

```diff
@@ -28,12 +28,33 @@ factory.define('facility')
   })
   .attr('name', ['city'], (city) => `${faker.address.cityPrefix()} ${city}`)
   .attr('organization', ['fromServer'], (fromServer) => {
-    return factory.build('organization', null, { fromServer });
+    return factory.build('organization', null, {
+      fromServer
+    });
   })
   .attr('organizationId', ['organization'], (organization) => organization.id)
   .attr('tags', ['id', 'fromServer'], (id, fromServer) => {
-    return times(faker.random.number({ min: 0, max: 5 }), () => {
-      return factory.build('facilityTag', { facilityId: id }, { fromServer });
+    return times(faker.random.number({
+      min: 0,
+      max: 5
+    }), () => {
+      return factory.build('facilityTag', {
+        facilityId: id
+      }, {
+        fromServer
+      });
+    });
+  })
+  .attr('facilitiesGroupings', ['id', 'fromServer'], (id, fromServer) => {
```

Comment:
This needs to be added down below as well inside that block of options. This will definitely break at least one test, which is good, since I wasn't sure how it was passing before.

Reply:
And hey, we're trying to be neat and alphabetized